cli/operator/node.go

```diff
@@ -187,6 +187,7 @@ var StartNodeCmd = &cobra.Command{
 			validation.WithMetrics(metricsReporter),
 			validation.WithDutyStore(dutyStore),
 			validation.WithOwnOperatorID(operatorData.ID),
+			validation.WithLimiter(validation.NewPeerRateLimitManager(20, 300000, cfg.P2pNetworkConfig.MaxPeers*2, 1*time.Minute)), // rate per second and duration
```

Comment:
300k is a reasonable limit for invalid messages? @MatheusFranco99 @MatusKysel

Reply:
@GalRogozinski please correct me if I'm wrong.
I think we selected 300k because the attack with such an amount didn't produce any impact on the network.
Nonetheless, it's way above any rate of messages expected. Should we decrease it? (I don't have yet a strong opinion on this trade-off)

Reply:
@moshe-blox 

@MatheusFranco99 and I concluded that if we find time to think about a lower bound we will change this.
But it isn't a priority and it is also ok to do it in a later PR